### DISCUSSION TITLE
default timezone not works

### DISCUSSION THREAD

**Discussion by chlgbdud** [nhn/tui.calendar#1298] [Q&A]
i set data

"start":"2022-10-10T09:00:00+09:00"

and

    timezone: {
      zones: [{
        timezoneName: 'Asia/Seoul',
      }, ],
    },

but in USA schedule created in 10/11.
countries without a day are created correct. what can i do?

**Comment by adhrinae** [comment]
I've tested it manually setting the timezone to Pacific Time and it works as expected.

![image](https://user-images.githubusercontent.com/14539203/193700221-f5743652-c6c1-452b-8cfb-6d1113852118.png)

More detailed information would be helpful to find out the problem such as:

- Calendar version you're using
- Browser
- OS

Especially you might need to implement the `customOffsetCalculator` option if you're running the calendar on IE.

https://github.com/nhn/tui.calendar/blob/main/docs/en/apis/options.md#timezone
